doc/source/NewSiteConfigs.rst

```diff
@@ -514,7 +514,7 @@ It is recommended to increase the stacksize limit by using ``ulimit -S -s unlimi
 11. Edit site config files and common config files, for example to remove duplicate versions of external packages that are unwanted, add specs in ``envs/unified-env.mylinux/spack.yaml``, etc.
 
 .. warning::
-   **Important:** Remove any external ``cmake@3.20`` package from ``envs/unified-env.mylinux/site/packages.yaml``. It is in fact recommended to remove all versions of ``cmake`` up to ``3.20``. Further, on Red Hat/CentOS, remove any external curl that might have been found.
+   Remove any external ``cmake@3.20`` package from ``envs/unified-env.mylinux/site/packages.yaml``. It is in fact recommended to remove all versions of ``cmake`` up to ``3.20``. Further, on Red Hat/CentOS, remove any external curl that might have been found.
```

Comment:
We should be able to use the new-ish
```
spack external find --scope system --exclude ... --exclude ...
```
syntax in the copy & paste instructions to achieve the same?

Reply:
updated